uwrt_arm_cartesian_controller/CMakeLists.txt

```diff
@@ -0,0 +1,97 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
This file should follow the template in the README(but I'm about to update it in #180 )